recipe/meta.yaml

```
@@ -3,7 +3,8 @@
 {% set sha256 = "62743265222a729c7fe94c40f7b90ccc1ac5568f5ee6df46884e7ce3c16c78c7" %}
 
 package:
-  name: {{ name|lower }}
+  # Setting a dummy name, because multi-output behaviour with similar name has surprising behaviour
+  name: {{ name|lower }}-dummy
```

```suggestion
  name: {{ name|lower }}-split
```
I think `-split` is more widely used in these cases.